package.json

```diff
@@ -48,6 +48,7 @@
 	},
 	"files": [
 		"dist",
+		"lib/**/*.d.ts",
```

Comment:
we should also include the "types" field in package.json

Reply:
Done.